test/src/main/java/org/apache/accumulo/test/functional/KerberosRenewalIT.java

```diff
@@ -200,7 +199,21 @@ private void createReadWriteDrop(AccumuloClient client) throws TableNotFoundExce
           new Key("a", "b", "c").compareTo(entry.getKey(), PartialKey.ROW_COLFAM_COLQUAL),
           "Did not find the expected key");
       assertEquals("d", entry.getValue().toString());
-      client.tableOperations().delete(table);
     }
+    client.tableOperations().delete(table);
+  }
+
+  private String createTableAndReturnTableName(AccumuloClient client) throws AccumuloException,
+      AccumuloSecurityException, TableNotFoundException, TableExistsException {
+    final String tableName = getUniqueNames(1)[0] + "_table";
+    try {
+      client.tableOperations().create(tableName);
+    } catch (TableExistsException e) {
+      log.debug("Table {} already exists. Deleting and trying again.", tableName);
+      client.tableOperations().delete(tableName);
```

Comment:
That's a good point, and looking at the stack trace, probably how this test failed in the first place.

> Another strategy may be to pull the create / delete into functions that retry and pause a little a few times.

This seems like a good idea. Also seems like a perfect place for the [`Wait.java`](https://github.com/cshannon/accumulo/blob/9d7a69d5ec4fb35c5f882a64596e7843ccbe886e/test/src/main/java/org/apache/accumulo/test/util/Wait.java#L28) test util that is part of #2799, added by @cshannon. I'm wondering if that could be broken into its own PR so it can be used elsewhere while #2799 is still under review.

Reply:
@EdColeman, this should be resolved as of [bdca3bd](https://github.com/apache/accumulo/pull/2902/commits/bdca3bde1fc9fef0034d9ebc2d1b6263f6d096d0).